tests/components/abode/test_init.py

```diff
@@ -0,0 +1,58 @@
+"""Tests for the Abode module."""
+from unittest.mock import patch
+
+from homeassistant.components.abode import (
+    DOMAIN as ABODE_DOMAIN,
+    SERVICE_CAPTURE_IMAGE,
+    SERVICE_SETTINGS,
+    SERVICE_TRIGGER_AUTOMATION,
+)
+from homeassistant.components.alarm_control_panel import DOMAIN as ALARM_DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.setup import async_setup_component
+
+from .common import setup_platform
+
+
+async def test_abode_setup_from_config(hass):
+    """Test setup from configuration yaml file."""
+    config = {
+        ABODE_DOMAIN: {
+            CONF_USERNAME: "user@email.com",
+            CONF_PASSWORD: "password",
+            "polling": True,
+        }
+    }
+    with patch("abodepy.utils.save_cache"):
+        assert await async_setup_component(hass, ABODE_DOMAIN, config)
```

Comment:
This is still creating abodepy_cache.pickle in the testing_config folder. I've tried patching everything I can think of, but no luck. Still need to figure this out then I'll make a new commit to delete the abodepy_cache.pickle.